configs/mmdet3d/monocular-detection/monocular-detection_tensorrt_dynamic-320x320-1600x1600.py

```diff
@@ -0,0 +1,16 @@
+_base_ = [
+    './monocular-detection_static.py', '../../_base_/backends/tensorrt.py'
```

Comment:
Why static?

Reply:
3D monocular tasks involve the transformation of image coordinates to camera coordinates, and in general the camera intrinsic matrix and image resolution are fixed.